test/request-data/resource.spec.js

```diff
@@ -22,7 +52,7 @@ describe('createResource()', () => {
       });
 
       it('passes the response and the resource to the callback', () => {
-        const projectData = testData.extendedProjects.createNew();
+        const projectData = testData.standardProjects.createNew();
```

Comment:
This is not the most important change in the world, but the request in this test doesn't request extended metadata, so it feels better to return a response without metadata. I based at least one of the new tests on this one, which is why I was looking at it.